app/index.html

```diff
@@ -13,15 +13,33 @@
     <meta name="description" content="" />
     <meta name="keywords" content="" />
 
-    <meta property="og:title" content="dGRANTS – xxx" />
-    <meta property="og:description" content="xxx" />
-    <meta property="og:image" content="dgrants_social_preview.png" />
-    <meta property="og:url" content="https://xxx" />
+    <meta
+      property="og:title"
+      content="dGRANTS – Gitcoin Decentralized Grants"
+    />
+    <meta
+      property="og:description"
+      content="The decentralized Grants product is Gitcoin’s first move towards decentralization. Our vision is that all of our main grant rounds will be completely community-curated and operated. This protocol is the first step to enabling the web3.0 community to manage how their public goods are funded"
```

Comment:
"all of the main grant rounds", since this text is from the DAO, not Gitcoin? Also usually I've seen it as "web3" instead of "web3.0"


```suggestion
      content="The decentralized Grants product is Gitcoin’s first move towards decentralization. Our vision is that all of the main grant rounds will be completely community-curated and operated. This protocol is the first step to enabling the web3 community to manage how their public goods are funded"
```